tools/mason/MasonEnv.chpl

```diff
@@ -43,6 +43,15 @@ proc MASON_CACHED_REGISTRY {
   return cachedRegistry;
 }
 
+proc MASON_OFFLINE {
+  const offlineEnv = getEnv('MASON_OFFLINE');
+  const default = 'false';
+
+  const offline = if offlineEnv == 'true' then offlineEnv else default;
```

Comment:
Some other cases to consider: maybe a user sets it to `TRUE` or `1` (true) or `0` (false)